bin/redeclipse.app/Contents/Info.plist

```diff
@@ -5,15 +5,15 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>English</string>
 	<key>CFBundleName</key>
-	<string>Red Eclipse Legacy</string>
+	<string>Blue Nebula</string>
```

Comment:
Can you rename `bin/redeclipse.app` to `bin/blue-nebula.app`?

Reply:
I have not touched this `.app` stuff intentionally, we're not using it at the moment anyway. We need to decide first how to proceed with packaging for macOS before we should touch those files IMO.